object/virtual_device_list.go

```diff
@@ -919,7 +919,13 @@ func (l VirtualDeviceList) Name(device types.BaseVirtualDevice) string {
 	dtype := l.Type(device)
 	switch dtype {
 	case DeviceTypeEthernet:
-		key = fmt.Sprintf("%d", UnitNumber-7)
+		// Ethernet devices of UnitNumber 7-19 are non-SRIOV. Ethernet devices of 
```

Comment:
I see, the linter is complaining because there's a trailing space at the end of this comment. Just need to remove that space and CI should pass

Reply:
doh! I couldn't tell from the linting error that this was the case. Sorry. Have pushed this change now